app/routes/events/components/EventAdministrate/AttendeeElements.tsx

```diff
@@ -66,28 +76,63 @@ export const PresenceIcons = ({ presence, registrationId }: PresenceProps) => {
   const dispatch = useAppDispatch();
 
   return (
-    <Flex justifyContent="center">
+    <Flex alignItems="center" justifyContent="center">
       <TooltipIcon
         content="Til stede"
-        iconClass={cx('fa fa-check', styles.greenIcon)}
+        iconName="checkmark"
+        iconClass={styles.greenIcon}
         transparent={presence !== 'PRESENT'}
         onClick={() =>
+          eventId &&
           dispatch(updatePresence(eventId, registrationId, 'PRESENT'))
         }
       />
+      <Tooltip content="For sent til stede">
+        <div
+          className={cx(
+            styles.tooLateIcon,
+            presence !== 'TOO_LATE' && styles.transparent
+          )}
+          onClick={() => {
+            eventId &&
+              dispatch(updatePresence(eventId, registrationId, 'TOO_LATE'));
+          }}
+        >
+          <svg
+            xmlns="http://www.w3.org/2000/svg"
+            width="22"
+            height="22"
+            viewBox="0 0 24 24"
+            fill="none"
+            stroke="var(--color-orange-6)"
+            strokeWidth="1.4"
+            strokeLinecap="round"
+            strokeLinejoin="round"
+          >
+            <path d="m12 10 2 4v3a1 1 0 0 0 1 1h2a1 1 0 0 0 1-1v-3a8 8 0 1 0-16 0v3a1 1 0 0 0 1 1h2a1 1 0 0 0 1-1v-3l2-4h4Z" />
+            <path d="M4.82 7.9 8 10" />
+            <path d="M15.18 7.9 12 10" />
+            <path d="M16.93 10H20a2 2 0 0 1 0 4H2" />
+          </svg>
+        </div>
+      </Tooltip>
       <TooltipIcon
         content="Ukjent"
-        iconClass={cx('fa fa-question-circle', styles.questionIcon)}
+        iconClass={styles.questionIcon}
+        iconName="help-outline"
         transparent={presence !== 'UNKNOWN'}
         onClick={() =>
+          eventId &&
           dispatch(updatePresence(eventId, registrationId, 'UNKNOWN'))
         }
       />
       <TooltipIcon
         content="Ikke til stede"
```

Comment:
```suggestion
        content="Ikke til stede (gir 2 prikker)"
```
Just to match @norbye's suggestion

Reply:
Should this say "Møtte ikke opp (gir 2 prikker)" - to match it even further?